app/assets/javascripts/common/reports.js

```diff
@@ -676,7 +682,9 @@ Reports = function ({
 
     const decemberKeys = dateKeysArray.filter((item) => item.includes("Dec"));
```

Comment:
Nitpick:  While naming a variable, we use the plural to indicate it's a collection. Instead of `dateKeysArray`, use  `dateKeys`.